cpg-language-go/src/test/resources/log4j2.xml

```diff
@@ -6,7 +6,7 @@
         </Console>
     </Appenders>
     <Loggers>
-        <Logger level="DEBUG" name="de.fraunhofer.aisec.cpg"/>
+        <Logger level="TRACE" name="de.fraunhofer.aisec.cpg"/>
```

Comment:
Comitting a log level change intentionally? Thinking about it: DEBUG prob is not a good log level for production

Reply:
Was committed unintentionally. It's the log level only for the tests, so DEBUG is fine here I think